### DISCUSSION TITLE
Nodal field question

### DISCUSSION THREAD

**Discussion by jessecarterMOOSE** [idaholab/moose#22885] [Q&A General] [answered]
Just a general finite element question, but maybe specifically for the MOOSE implementation. 

Let's say I have an auxiliary field variable, like temperature, defined at nodes. In a finite deformation mechanics model, do the values of my field variables interpolated to the quadrature points change/update on nonlinear iterations? The nodal values are not changing but the displacements, and therefore the shape of the element, can change as the step converges.

**Comment by hugary1995** [comment, marked as answer]
In MOOSE/LibMesh, Nodal variables and their interpolants do not change with mesh deformation (on the same tessellation). Their gradients do.

In a more general setting of FEM, interpolants could change if you use some less common parametric mapping for element coordinates. But I've never seen that in MOOSE.

**Reply by jessecarterMOOSE** [reply]
ok, so shouldn't matter if I'm using displaced or undisplaed mesh to get the interpolated values at the qp's?

**Reply by hugary1995** [reply]
Yep. It doesn't matter.

**Reply by jessecarterMOOSE** [reply]
:+1:
